files/build/Makefile

```diff
@@ -0,0 +1,37 @@
+SONIC_BUILD_TOOLS = sonic-build-tools
```

Comment:
>sonic-build-tools [](start = 20, length = 17)

The name is already taken by [https://github.com/Azure/sonic-build-tools](https://github.com/Azure/sonic-build-tools)
Better to move into src folder.

Reply:
Sure, I will move it to src folder

---
In reply to: [537937333](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r537937333) [](ancestors = 537937333)